app/components/participant-creator/component.js

```diff
@@ -32,13 +32,20 @@ export default Ember.Component.extend({
     _generate(batchSize, tag) {
         var ret = [];
         for (let i = 0; i < batchSize; i++) {
-            ret.push(`${makeId(5)}${tag ? `-${tag}` : ''}`);
+            var generate = true;
+            while (generate) {
+                var id = `${makeId(5)}${tag ? `-${tag}` : ''}`;
+                if (!ret.contains(id) || !this.get('store').peekRecord('account', id)) {
```

Comment:
Yeah, that would be a lot of network requests. Is there a way to reload the store each time accounts are created?